### ISSUE TITLE
Edit button in toolbar

### ISSUE THREAD

**Issue by Crivens** [d2phap/ImageGlass#665]
### Is your feature request related to a problem? Please describe.
Currently if you want to use the edit image link to an external editor, then you need to right click and use the context menu, which seems like an unnecessary step when the toolbar exists

### Describe the solution you'd like
A new icon on the toolbar to access the edit image option, or even an option to allow a single click (or double click) on the image itself to launch the external editor

**Comment by d2phap** [comment]
Thanks @Crivens 
It's duplication of #641, and also implemented in IG Moon: https://imageglass.org/moon

**Comment by Crivens** [comment]
Sorry didn’t see that when I searched for it!

Also feature #413 is outstanding for more info. Any chance it will be implemented (ie. kill off ImageGlass as soon as the external editor is launched)? Sounds daft, but is very niggly and really this one thing can almost make me go back to my own personal image previewer (would rather use yours than find the time to develop mine a bit more) that I used to use solely for my own work projects.
